cmd/dummy-handlers.go

```diff
@@ -95,25 +96,29 @@ func (api objectAPIHandlers) DeleteBucketWebsiteHandler(w http.ResponseWriter, r
 	w.(http.Flusher).Flush()
 }
 
+type allowedMethod string
+
+// Define strings
+const (
+	GET    allowedMethod = http.MethodGet
+	PUT    allowedMethod = http.MethodPut
+	HEAD   allowedMethod = http.MethodHead
+	POST   allowedMethod = http.MethodPost
+	DELETE allowedMethod = http.MethodDelete
+)
+
 // GetBucketCorsHandler - GET bucket cors, a dummy api
 func (api objectAPIHandlers) GetBucketCorsHandler(w http.ResponseWriter, r *http.Request) {
 	ctx := newContext(r, w, "GetBucketCorsHandler")
 
-	type allowedMethod int
-	const (
-		GET allowedMethod = iota
-		PUT
-		HEAD
-		POST
-		DELETE
-	)
 	type corsRule struct {
-		AllowedMethod allowedMethod `xml:"AllowedMethod"`
-		AllowedOrigin string        `xml:"AllowedOrigin"`
-		ExposeHeader  string        `xml:"ExposeHeader"`
-		ID            string        `xml:"ID"`
-		MaxAgeSeconds int64         `xml:"MaxAgeSeconds"`
+		AllowedHeders  []string        `xml:"AllowedHeaders"`
+		AllowedMethods []allowedMethod `xml:"AllowedMethod"`
+		AllowedOrigins []string        `xml:"AllowedOrigin"`
+		ExposeHeaders  []string        `xml:"ExposeHeader"`
+		MaxAgeSeconds  int64           `xml:"MaxAgeSeconds"`
```

Comment:
Even though `ID` is optional should we still keep it here, just for completeness sake? We are not filling any of these other elements also?

Reply:
There is no ID in official docs for Cors.